### PR TITLE
Simple Series info page

### PR DESCRIPTION
**Pull request by ptitdam2001**

### CODE REVIEW

File: client/web/src/components/series/Info/SeriesInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      />
+      <Box p={1}>
+        {series ? (
+          <Paper>
+            <Box p={1}>
+              <p>id: {series.id}</p>
+              <p>description: {series.seriesDescription}</p>
+              <p>modality: {series.modality}</p>
+              <p>
+                Nb of Series related instances:{' '}
```
Comment (cquennouelle) [inline client/web/src/components/series/Info/SeriesInfo.tsx]: To translate?

File: client/web/src/components/common/Toolbar/Toolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode, Fragment } from 'react'
```
Comment (cquennouelle) [inline client/web/src/components/common/Toolbar/Toolbar.tsx:1]: For a newbie reviewer, some comments might be helpful.

File: client/web/src/pages/About.tsx [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+  const { t } = useTranslation()
 
   return (
-    <Fragment>
-      <h1>About</h1>
+    <>
+      <Toolbar label={t('About')} />
+
       <p>
         Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas
```
Comment (cquennouelle) [inline client/web/src/pages/About.tsx:13]: One day, one will change this text. I know that.

File: client/web/src/components/common/Toolbar/Toolbar.style.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { createStyles, Theme } from '@material-ui/core'
+import { Theme, createStyles } from '@material-ui/core'
```
Comment (bat3) [inline client/web/src/components/common/Toolbar/Toolbar.style.ts:1]: background: theme.palette.primary.main,
The red color... make me afraid

File: client/web/src/features/series/seriesSaga.ts [hunk 5]
```diff
@@ -12,6 +12,13 @@ export function* fetchSeries() {
   })
 }
 
+export function* fetchOneSeries(action: FetchOneSeriesAction) {
+  const { id } = action
+  yield put({ type: actionTypes.UNSET_CURRENT_SERIES })
```
Comment (bat3) [inline client/web/src/features/series/seriesSaga.ts:17]: Why unset current series ?

File: client/web/src/components/series/Info/SeriesInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+) => {
+  const classes = useStyle()
+  const { id } = props
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const series: ISeries = useSelector(selectors.getCurrent)
+  const { t } = useTranslation()
+
+  useEffect(() => {
+    dispatch({ type: actionTypes.FETCH_ONE_SERIES, id })
```
Comment (bat3) [inline client/web/src/components/series/Info/SeriesInfo.tsx:31]: I don't see the need of fetch the series ? all information are in const series: ISeries = useSelector(selectors.getCurrent) no ?

**Review by bat3 — CHANGES_REQUESTED** [review]

File: client/web/src/features/series/seriesSaga.ts (hunk 5 above)
Comment (ptitdam2001) [reply client/web/src/features/series/seriesSaga.ts:17]: if there is one error on fetch (not found for example), I don't want to show the previous selected series

File: client/web/src/components/series/Info/SeriesInfo.tsx (hunk 6 above)
Comment (ptitdam2001) [reply client/web/src/components/series/Info/SeriesInfo.tsx:31]: useSelector(selectors.getCurrent) returns the last result of fetch the series. It is better to call a fetch for having the last fresh data.

File: client/web/src/components/common/Toolbar/Toolbar.style.ts (hunk 4 above)
Comment (ptitdam2001) [reply client/web/src/components/common/Toolbar/Toolbar.style.ts:1]: we have not configure one main theme, it is the default theme

Comment (ptitdam2001) [reply client/web/src/components/common/Toolbar/Toolbar.style.ts:1]: theme.palette.primary.main is blue used by the application navbar

Comment (ptitdam2001) [reply client/web/src/components/common/Toolbar/Toolbar.style.ts:1]: I have created a new task about providing a theme

**Review by bat3 — APPROVED** [review]

File: client/web/src/components/series/Info/SeriesInfo.tsx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+                <span>{series.seriesInstanceUID}</span>
+              </p>
+              <p>
+                <span>{t('Body part examined')}: </span>
+                <span>{series.bodyPartExamined}</span>
+              </p>
+              <Divider />
+              <div>{t('Related labels')}</div>
+              <Box display="flex" flexDirection="row" p={1}>
+                {series.labels.map((label: ILabel) => (
```
Comment (bat3) [inline client/web/src/components/series/Info/SeriesInfo.tsx]:
```suggestion
                {series.labels && series.labels.map((label: ILabel) => (
```

**Review by bat3 — CHANGES_REQUESTED** [review]

**Review by bat3 — APPROVED** [review]

**Pull request merged**